js/apps/admin-ui/src/clients/authorization/policy/Regex.tsx

```diff
@@ -54,6 +55,34 @@ export const Regex = () => {
           {...register("pattern", { required: true })}
         />
       </FormGroup>
+      <FormGroup
+        label={t("targetContextAttributes")}
+        fieldId="pattern"
+        labelIcon={
+          <HelpItem
+            helpText={t("targetContextAttributesHelp")}
+            fieldLabelId="clients:targetContextAttributes"
+          />
+        }
+        isRequired
+        validated={errors.pattern ? "error" : "default"}
+        helperTextInvalid={t("required")}
+      >
+        <Controller
+          name="targetContextAttributes"
+          defaultValue={false}
+          control={control}
+          render={({ field }) => (
+            <Checkbox
+              id="required"
+              data-testid="standard"
```

Comment:
I don't believe this test id is used anywhere, but the name should probably match the conventions of the rest of the component.

Reply:
removed